grails-app/views/project/index.gsp

```diff
@@ -265,19 +267,20 @@
                         </cl:ifAdmin>
                     </cl:isLoggedIn>
                     <cl:ifValidator project="${projectInstance}">
-                    <g:link class="btn btn-default btn-hollow grey" controller="task" action="projectAdmin"
-                            id="${projectInstance.id}">Validate tasks</g:link>
-                </cl:ifValidator>
-
+                        <g:link class="btn btn-default btn-hollow grey" controller="task" action="projectAdmin"
+                                id="${projectInstance.id}">Validate tasks</g:link>
+                    </cl:ifValidator>
                 </div>
             </div>
         </div>
 
-        %{--<div class="row">--}%
-            %{--<div class="col-sm-12 image-origin float-right">--}%
-                %{--<p>Image by [Name] from Flickr</p>--}%
-            %{--</div>--}%
-        %{--</div>--}%
+        <g:if test="${projectInstance.backgroundImage && projectInstance.backgroundImageAttribution}">
```

Comment:
IMO that we need to include the `div class="image-origin"` below even without the attribution as it has a large margin that shows off the whole hero image...  Obviously the <p> shouldn't be displayed if no backgroundImageAttribution is available.